pkg/notify/notifier/sms/huawei.go

```diff
@@ -113,9 +114,38 @@ func (h *HuaweiNotifier) MakeRequest(ctx context.Context, messages string) error
 func buildWSSEHeader(appKey string, appSecret string) string {
 	now := time.Now().Format("2006-01-02T15:04:05Z")
 	nonce := strings.ReplaceAll(string(uuid.NewUUID()), "-", "")
-	sha256 := getSha256Code(nonce + now + appSecret)
+	sha256 := getSha256Code(nonce + now + strings.ReplaceAll(appSecret, "\n", ""))
 	passwordDigest := base64.StdEncoding.EncodeToString([]byte(sha256))
-	return fmt.Sprintf(`"UsernameToken Username="%s",PasswordDigest="%s",Nonce="%s",Created="%s"`, appKey, passwordDigest, nonce, now)
+	return fmt.Sprintf(`UsernameToken Username="%s",PasswordDigest="%s",Nonce="%s",Created="%s"`, strings.ReplaceAll(appKey, "\n", ""), passwordDigest, nonce, now)
+}
+
+// Note here, the keys corresponding to the return values are alertname,alerttype,severity,message,summary.
+// Pls make your custem SMS template containing five placeholders, you can set the template on the console of SMS huawei cloud provider.
```

Comment:
you can set the template on the console of SMS huawei cloud provider. =>
you can create an SMS template in Huawei Cloud's SMS console.